src/main/java/core/basesyntax/model/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.model;
+
+public class RightTriangle extends Color {
```

Comment:
The name 'RightTriangle' is appropriate for a class representing a triangle. However, ensure that your class names do not imply actions or verbs which are better suited for methods.